webapp/src/components/boardTemplateSelector/boardTemplateSelector.tsx

```diff
@@ -98,7 +98,8 @@ const BoardTemplateSelector = (props: Props) => {
     }
 
     const handleUseTemplate = async () => {
-        await mutator.addBoardFromTemplate(currentTeam?.id || Constants.globalTeamId, intl, showBoard, () => showBoard(currentBoardId), activeTemplate.id, currentTeam?.id)
+        const templateID: string = activeTemplate.properties.templateID as string
+        await mutator.addBoardFromTemplate(currentTeam?.id || Constants.globalTeamId, intl, showBoard, () => showBoard(currentBoardId), templateID, currentTeam?.id)
```

Comment:
I think this is not what I expect. I understand that from one side we have the `template.id` that refers the id in the instance, and we should have a custom `trackingTemplateId` or something like that, that is only set for the built-in templates. And then, use that whenever we are tracking that information. So, the trackingID is only used for tracking purposes, not for using the template in any way.

Reply:
Done.

Reply:
Isn't this going to pollute telemetry with the template ID of every board created from templates, not just the default (built-in) templates?  I don't think the requirement is to track user generated templates.

Reply:
@wiggin77 I see your point.

@wuwinson do we wanna track user-generated templates? I'm thinking that we should so we know how much users use the board template feature.